Update_user.php

```diff
@@ -21,20 +32,20 @@
 				<a href = "01_login_real.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
 			<?php
-				//Pulls the links from the nav_login.php page and places them in the navigation div
-				require 'nav_login.php'; //'require' is 100% needed for the site to run 
+				//Pulls the links from the nav_admin.php page and places them in the navigation div
```

Comment:
This is the update user page for admin so In need to change to admin nav beucase if it will be normal nav(user nav) then I can't access to the setting.